share/availability/discovery/discovery.go

```diff
@@ -107,7 +108,7 @@ func (d *Discovery) EnsurePeers(ctx context.Context) {
 		return
 	}
 	// subscribe on Event Bus in order to catch disconnected peers and restart the discovery
-	sub, err := d.host.EventBus().Subscribe(&event.EvtPeerConnectednessChanged{})
+	sub, err := d.host.EventBus().Subscribe(&event.EvtPeerConnectednessChanged{}, eventbus.BufSize(512))
```

Comment:
let's extract into a var in this file - Can we also document that the default buf size is 16 and why we increased it to 512

Reply:
512 is overkill btw, and 256 is more than enough, or even 128. There is literally nothing that could block event reading routine anymore